src/odc/tools/TestInt16_MissingCodec.cc

```diff
@@ -69,8 +69,8 @@ class MockReaderIterator3
 				data_ = columns_[0]->coder().missingValue();
 				break;
 			default:
```

Comment:
Can we fix the indenting?

Reply:
Sure, how about I also add clang format on top? (followup commit)

Reply:
Sure, how about I also add clang format on top? (followup commit)